### PR TITLE
chore: various eslint fixes

### PR DESCRIPTION
**Pull request by dbrugger**
### Affected issues

- Contributes to #973 

### Proposed changes

- ErrorPage: Remove unused `labels` prop
- ScrollGradient, Tearsheet, Wizard: mark some vars as throw away

### Testing instructions

-

### CODE REVIEW

File: src/components/Tearsheet/Tearsheet.js [hunk 1]
```diff
@@ -80,8 +79,8 @@ class Tearsheet extends Component {
       },
       labels,
       loading,
-      loadingMessage,
-      isOpen,
+      loadingMessage: _, // trow away
```
Comment (paul-balchin-ibm) [inline src/components/Tearsheet/Tearsheet.js:82]: "throw"

**Review by paul-balchin-ibm — APPROVED** [review]
Only found a spelling error

File: src/components/Tearsheet/Tearsheet.js (hunk 1 above)
Comment (dbrugger) [reply src/components/Tearsheet/Tearsheet.js]: nice catch thanks @paul-balchin-ibm !

**Pull request merged**